### PR TITLE
Readme change

### PR DESCRIPTION
**Pull request by ericadohring**

### CODE REVIEW

**Comment by maliksalman** [comment]
This looks good

**Review by jeffellin — CHANGES_REQUESTED** [review]
This stinks I like lowercase

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# git-practice
+# Git Practice
```
Comment (maliksalman) [inline README.md:1]: Can you make it `GIT`

**Comment by DustyP53** [comment]
Works for me, but was this supposed to be all caps?    ;)

**Review by swathikaThiyagarajan — APPROVED** [review]
Looks good - Swathi

File: README.md (hunk 1 above)
Comment (ericadohring) [reply README.md:1]: Let's do this next time. It's time to go to prod.

**Pull request merged**